src/modules/src/Kconfig

```diff
@@ -59,3 +59,49 @@ config ESTIMATOR_OOT
 	default n
 
 endmenu
+
+menu "Motor configuration"
+
+config MOTORS_START_DISARMED
+    bool "Set disarmed state after boot"
+    default n
+    help
+      When enabled, the firmware will boot in disarmed state and one needs to
+      arm the drone explicitly before starting the motors
+
+config MOTORS_DEFAULT_IDLE_THRUST
+    int "Default idle thrust for motors in armed state"
+    range 0 65535
+    default 0
+    depends on MOTORS_START_DISARMED
+    help
+      Default thrust for motors when idling in armed state, expressed as an
+      integer in the range 0 to 65535. This can be overridden with parameters;
+	  the value specified here applies to the case when the persistent storage
```

Comment:
Some indent shenanigans here.

Reply:
Yep, I wanted to ask about indentation policy. Some Kconfig files use mixed 2-space vs 4-space indenting, and I've also seen a few tabs occasionally here and there. I can re-tab the whole file to use spaces in this PR if that's okay, and then I'll also switch to consistent 4-space indentation.

Reply:
That is ok, thanks.

Reply:
All Kconfig files were now reformatted to use 4-space indentation consistently.